src/Narochno.Primitives/Parsing/Convert.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace Narochno.Primitives.Parsing
+{
+    public static class Convert<TType>
+        where TType : struct
+    {
+
+        private static IParser<TType> parser;
+
+        static Convert()
+        {
+            Refresh();
+        }
+
+        /// <summary>
+        /// Refreshes the parser from the DefaultParserLibrary if it is changed.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static void Refresh()
+        {
+            parser = DefaultParserLibrary.Instance.GetParser(typeof(TType)) as IParser<TType>;
```

Comment:
I think this is a little overkill, the StringExtensions inside the parser library just call this each time when they are run and that seems performant enough: https://github.com/Narochno/Narochno.Primitives/blob/master/src/Narochno.Primitives/Parsing/StringExtensions.cs#L16

This is making things more complex than they need to be imo, do you have perf data with/without the cache?